src/scripts/templates/standalone/Traverse sites tree.js

```diff
@@ -3,6 +3,10 @@
 // Standalone scripts have no template.
 // They are only evaluated when you run them. 
 
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
+
```

Comment:
K, will figure out something for this one. Can probably do more specific reassignment of the functions in order to handle both.